projectDocs/translating/crowdin.md

```diff
@@ -1,45 +1,91 @@
+# Translating with Crowdin
 
-Crowdin is used to translate the main NVDA interface and user documentation.
+Crowdin is used to manage the translations of the main NVDA interface and user documentation.
 NVDA's Crowdin project: <https://crowdin.com/project/nvda>.
 
-This document covers setting up a Crowdin account, connecting it with PoEdit, and translating the main interface and user documentation using Crowdin and PoEdit.
+This document covers setting up a Crowdin account, and translating the main interface and user documentation using either the Crowdin web interface or Poedit / nvdaL10nUtil.
 
-## Setup
-
-### Create Crowdin account
+## Creating a Crowdin account
 
 1. Create a [Crowdin account](https://accounts.crowdin.com/register?continue=%2Fproject%2Fnvda).
 1. Message the [translators mailing list](https://groups.io/g/nvda-translations) or <info@nvaccess.org> to request being added as a translator.
 Please include your Crowdin username and the languages you wish to translate.
 
-### Setup PoEdit
+## Translation workflows
+
+There are 2 common workflows for translating with Crowdin:
+
+1. Translating strings directly via Crowdin's web interface. Or
+1. Downloading a translation file from Crowdin, translating with Poedit, and uploading the file back to Crowdin.
+
+### Translating via Crowdin's web interface
+
+For instructions on how to translate strings using Crowdin's web interface, please read [Crowdin's documentation for translators](https://support.crowdin.com/for-translators/).
+
+Note: If you are a screen reader user, you may find Crowdin's web interface inefficient or impossible to use.
+Therefore you may choose to use the Poedit workflow instead.
+
+#### Translation Reviews / Approvals
+
+Due to accessibility issues and to be able to support the alternative workflow using Poedit, the translation approvals feature has been disabled in the NVDA project on Crowdin.
+This means that all new strings are essentially auto approved.
+However, to maintain quality control, only translators specifically added to the project can add or change strings.
+
+### Translating with Poedit
+
+Poedit is a desktop application which is commonly used to translate files.
+It is fairly accessible and is used by many blind and vision impaired translators.
+
+The workflow for translating with Poedit involves:
+1. Downloading the translation file using NVDA's l10n utility
+1. Opening the translation file in Poedit, translating one or more strings, and saving the file.
+1. Uploading the translation file back to Crowdin.
+
+Warning: Do not download / upload translation files via Crowdin's web interface, nor use Poedit's Crowdin cloud translation feature, as these methods have been found to corrupt xliff files.
+Please instead use NvDA's l10n utility for download and upload.
+NVDA's l10n utility will automatically use the appropriate Crowdin settings when downloading and uploading, and will detect and correct corruptions in xliff files.
```

Comment:
Is it worth indicating here the required settings to avoid corruption? Or in an l10nUtils dev documentation?
E.g. auto-approval, how to handle blank or identical strings, what does apply to xliff files only, etc...

To summarize, have a summary of the input you have taken when creating l10nUtils in case you are not available anymore when nneded to explain them, so that we do not turn back in a translation framework hard to understand as mrconfig had become.

Reply:
I think it would be good to mention it somewhere, but I feel this document is not the place and should not hold up this pr.
For the record:
*Download is just a normal Crowdin export, but once downloaded the file is preprocessed to remove corrupt translations such as where the literal string is `<target />` or `<target></target>` or `&lt;target&gt;&lt;/target&gt;` or `&lt;target /&gt;`.
* Upload  first strips the xliff of any notes, empty strings, or corrupt strings (see above). And then Imports to Crowdin  with settings of autoApprove, and importEqSuggestsions (translation can match source), stripping first avoids the issue where this would simply translate all up to 100% with the original strings.

Reply:
I agree that we do not need to hold off this PR due to these explanations. Could you please open a dedicated issue to have this work tracked?